users/vieting/experiments/switchboard/ctc/feat/network_helpers/perturbation.py

```diff
@@ -1,4 +1,5 @@
 from typing import List, Dict, Optional, Any
+import inspect
```

Comment:
Also here, these changes are not really what the title suggests this PR is about. Please split this up so that we can also have a somewhat helpful git history.